src/modules/ShortcutGuide/ShortcutGuide.Ui/Program.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Windows;
+using ManagedCommon;
+using Microsoft.UI.Dispatching;
+using Microsoft.Windows.AppLifecycle;
+using ShortcutGuide.Helpers;
+using Application = Microsoft.UI.Xaml.Application;
+
+namespace ShortcutGuide
+{
+    public sealed class Program
+    {
+        private static readonly string[] InbuiltManifestFiles = [
+            "+WindowsNT.Shell.en-US.yml",
+            "+WindowsNT.WindowsExplorer.en-US.yml",
+            "+WindowsNT.Notepad.en-US.yml",
+            "Microsoft.PowerToys.en-US.yml",
+        ];
+
+        [STAThread]
+        public static void Main()
+        {
+            if (PowerToys.GPOWrapper.GPOWrapper.GetConfiguredShortcutGuideEnabledValue() == PowerToys.GPOWrapper.GpoRuleConfigured.Disabled)
+            {
+                Logger.LogWarning("Tried to start with a GPO policy setting the utility to always be disabled. Please contact your systems administrator.");
+                return;
+            }
+
+            if (!Directory.Exists(ManifestInterpreter.PathOfManifestFiles))
+            {
+                Directory.CreateDirectory(ManifestInterpreter.PathOfManifestFiles);
```

Comment:
You can remove the `if` statement, because CreateDirectory does that if the location doesn't exist.